File: Arreglos/Java/Filas_D_S.java
package Java;
public class Filas_D_S {

    public static void main (String[] args){
        int dimension = 5;
        int arreglo[][]= new int [dimension][dimension];
        int filas=0, columnas=0;
    
        for (filas=0; filas<dimension; filas++){
            for (columnas=0;columnas<dimension; columnas ++){
                if (columnas<filas){
                    System.out.print("      ");
                }else{
                    System.out.print("("+filas+","+columnas+") ");
                }
                
             }   
        System.out.println(" "); 
        }
    }
    
}

File: Arreglos/Python/Filas_D_S.py
'''
Author: "Space Team"
Date: 12/02/2022
Description: Printing the upper diagonal of a matrix traversed by rows
'''

#Input
range = [0,1,2,3,4]
filas = 0
columnas = 0

#Algorithm
for filas in range:
    for columnas in range:
        if columnas < filas:
            print("      ", end= "")
            
    for columnas in range:
        if columnas >= filas:
            print(f"({filas},{columnas}) ", end= "")
    print(" ")   

#Output        


